behat/features/tabs.feature

```diff
@@ -0,0 +1,144 @@
+Feature: Book page
+  As an anonymous user
+  We need to be able to see the tabs list in the book page.
+
+  @api
+  Scenario Outline: Visit a books page
+    Given I am an anonymous user
+    When  I visit "books/זה-ספר-בית-המדות"
+    Then  I should see the text "<tab>"
+    And   I should see the text "<text>"
+
+  Examples:
+    | tab                         | text                                    |
+    | Book producers              | מרגליות, יהודה ליב בן אשר                  |
+    | Book structure              | על חלק דברי המוסר                        |
+    | Textual models              | מרגליות                                   |
+    | Type of book                | Title of the book in Latin characters    |
+    | Publisher/ Printing press   | Place of publication as it appears in    |
+    | Persons mentioned in book   | שיחיו ליום הולדתו בשנת                      |
+    | Topics                      | Topic                                    |
+    | Target audience             | Target audience as described in the book |
+    | Catalog numbers             | University catalog                       |
+    | References and bibliography | לאומית                                    |
+
+
+  @api
+  Scenario Outline: Visit a books page
```

Comment:
(here for example the ``Scenario Outline`` is fine since we need to start every time with the same URL.